README.rst

```diff
@@ -244,8 +244,8 @@ We can then start its components (note the runtime arguments, as described above
 
 You can also use the CDAP CLI to start the Flow and Service::
 
-  $ cdap-cli.sh start flow KafkaIngestionApp.KafkaIngestionFlow \'"kafka.zookeeper"="localhost:2181", "kafka.topic"="MyTopic"\'  
-  $ cdap-cli.sh start service KafkaIngestionApp.KafkaStatsService
+  $ cdap cli start flow KafkaIngestionApp.KafkaIngestionFlow \'"kafka.zookeeper"="localhost:2181", "kafka.topic"="MyTopic"\'  
```

Comment:
It might work; I haven't tried it. But I notice that you didn't escape the leading and trailing single-quotes, which I think is required.